tests/common/helpers/enable-dynamic-buffer.py

```diff
@@ -131,6 +136,28 @@ def stop_traditional_buffer_model(config_db):
     return lossless_pgs
 
 
+def check_buffermgrd_is_running():
+    cmd_get_buffermgrd_status = "docker exec swss supervisorctl status buffermgrd"
+    max_try_times = 10
+    try_times = 0
+    while try_times < max_try_times:
+        try_times += 1
+        proc = subprocess.Popen(cmd_get_buffermgrd_status, shell=True, stdout=subprocess.PIPE)
+        output, err = proc.communicate()
+        if err:
+            logger.log_notice("Failed to check buffermgrd status: {}".format(err))
```

Comment:
Does this require retry? If so, please add retry_timer to the log like below log message.